Mykola_Horetskyi/1/Bob.py

```diff
@@ -0,0 +1,18 @@
+text = input('input text:\n')
+
+text = text.strip()
+
+if len(text) == 0:
+    print( "Fine. Be that way!")
+else:
+    isQuestion = text[len(text) - 1] == '?'
```

Comment:
You can use method endswith() of [-1] index to get the last element of the sequence